apps/web/src/pages/liquidity/[tokenId].tsx

```diff
@@ -361,9 +361,9 @@ export default function PoolPage() {
   ])
 
   const owner = useSingleCallResult({
-    contract: tokenId ? positionManager : null,
+    contract: tokenId && positionManager ? positionManager : undefined,
     functionName: 'ownerOf',
-    args: useMemo(() => [tokenId] as const, [tokenId]),
+    args: useMemo(() => (tokenId ? [tokenId] : []) as [bigint], [tokenId]),
```

Comment:
if we're fixing typing issue, can we not change the original behavior and simply add the type assertion?
```suggestion
    args: useMemo(() => [tokenId] as [bigint], [tokenId]),
```